contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,141 @@
+### They’re flexible
+
+We’d rather hire new people than keep moving people around to fill gaps. That said, we’re happy for people to move between teams when needed, ideally no more often than every three to nine months. There are two scenarios that could trigger a move:
+
+* A small team may realize they no longer need someone, or that they could use the help of someone internally on another small team.
+* Someone might want to move to another small team to develop their skills or experience.
+
+Rather than the team lead making the call, it’s always the manager who decides if one of their direct reports should move between teams. But, once in a small team, you’re only in one team. Someone being in multiple small teams at once defeats the purpose of ownership.
+
+If someone feels the need to be in more than one small team, that likely means we need to hire. That can mean we’re ‘overstaffed’ at times, but ultimately good people will do good work to push our product forward. And it’s better to have slightly overstaffed small teams than to be perpetually understaffed and for your product to suffer as a result.
+
+Small teams are also able to work with other small teams on multidisciplinary projects and members of one small team can and should attend meetings on other small teams to help with collaboration if needed.
+
+### 
+
+### The tradeoffs of small teams
+
+Our small teams structure is a success because we work hard on making it one. There are tradeoffs. 
+
+Working as a series of startups within a startup risks creating silos and there could be whole teams or individual team members who drift away from our mission or feel disconnected from the company. 
+
+To help alleviate this, all of our small teams have annual off-sites in addition to our big company get-together. Everyone also has a generous quarterly travel budget and people are encouraged to go work with colleagues either on the same small team as them or in another part of the company. We mix things up and spend time together IRL to work on problems and get things shipped.
+
+Small teams also require that everyone is very good at written communication, be it a Slack message, or a Github pull request or issue. This helps everyone at the company to both see and understand what's being worked on and why.
+
+You also need to hire the right people to work on small teams. Not everyone will embrace this way of working. It requires people who take extreme ownership of ideas, are self-starting, authentic, and low-ego. We call them cracked engineers. And we have some [strong thoughts](https://newsletter.posthog.com/p/hiring-and-managing-cracked-engineers) on what makes an engineer cracked.
```

Comment:
```suggestion
### The tradeoffs of small teams

Small teams aren’t perfect. And sometimes, to make them work, we’ve had to create workarounds and accept some tradeoffs.

- **They clash.** Our small teams do sometimes clash, especially when more than one team is working on the same product. We mitigate this by being aggressively transparent. We also have a list of everything we do [and who owns it](/handbook/engineering/feature-ownership). This helps avoid clashes by clearly establishing feature ownership.  

- **Fuzzy ownership.** This isn’t unique to small teams, but we still mitigate against it. If a product or project doesn't have a clear owner, our [fuzzy ownership process](/handbook/company/fuzzy-ownership) encourages people to create a PR and find a solution. We also [actively encourage that](https://posthog.com/handbook/values) people to step on toes in a low-ego way. 

- **Speed over seamlessness.** Small teams allow us to ship fast. But the tradeoff here is that we tolerate some level of not-so-great integration in exchange for speed. This isn't a problem unique to PostHog. AWS is a great example of a very, very big company that favours speed over integration.

- **Add more small teams!** We want to avoid having big, slow teams. In doing so, we’ve kept adding more small teams. This is a hypothetical concern, but creating too many small teams could bottleneck each other. 

- **You’ve got to hire right.** Not everyone will embrace this way of working. It requires people who take extreme ownership of ideas, are self-starting, authentic, and low-ego. We call them cracked engineers. And we have some [strong thoughts](https://newsletter.posthog.com/p/hiring-and-managing-cracked-engineers) on what makes an engineer cracked.

```